ext/readline/readline_cli.c

```diff
@@ -349,9 +349,14 @@ static int cli_is_valid_code(char *code, size_t len, zend_string **prompt) /* {{
 			case heredoc:
 				if (code[i - (heredoc_len + 1)] == '\n' && !strncmp(code + i - heredoc_len, heredoc_tag, heredoc_len) && code[i] == '\n') {
```

Comment:
Can we drop this case?

Reply:
Good catch, thanks! Indeed, this is now superfluous.